decentraland/components/presentational/LandForSaleInfo.js

```diff
@@ -78,5 +78,6 @@ const styles = StyleSheet.create({
     fontSize: responsiveFontSize(4),
     fontWeight: "bold",
     color: Colors.assetInfoText,
+    paddingTop: responsiveHeight(0.2),
```

Comment:
Hmm, I think we need more than this still

Reply:
Ooh wait, never mind. My pull of the latest changes didn't work.

Reply:
I think 0.4 looks the best locally

Reply:
```suggestion
    paddingTop: responsiveHeight(0.4),
```

Reply:
![TempAlignment](https://user-images.githubusercontent.com/2731712/55423884-89d3b180-5544-11e9-9614-b94e88ba3a1b.png)